spec/README.mediawiki

```diff
@@ -1130,30 +1152,99 @@ DEX will send a list of '''Match objects'''.
 |-
 | rate      || int    || the rate that the order matched at (as quote asset per unit base asset, atoms)
 |-
-| address   || string || the client's receiving address
+| address   || string || the counterparty's receiving address
 |-
-| timestamp || int    || UTC timestamp. order broadcast time
+| sig       || string || DEX's hex-encoded signature of the serialized notification data. serialization described below
+|}
+
+'''Match notification serialization'''
+
+{|
+! field      !! size (bytes)  !! description
+|-
+| orderid    || 32 || the order ID
+|-
+| matchid    || 8  || the ID assigned to this match
+|-
+| quantity   || 8  || the matched amount, in atoms of the base asset
+|-
+| rate       || 8  || the rate that the order matched at
+|-
+| timestamp  || 8  || the client's UNIX timestamp
+|-
+| address    || varies || ASCII encoded receiving address for the match
```

Comment:
Just occurred to me that ASCII may not be the best choice here, although I imaging the character set is more than adequate.  With the base58 encoding, there shouldn't be any difference between ASCII and UTF-8 e.g., but I'm not certain.